rosetta-source/src/main/resources/result-json-files/fpml-5-13/products/variance-swaps/eqvs-ex06-variance-option-transaction-supplement.json

```diff
@@ -1,159 +1,111 @@
 {
   "trade" : {
-    "tradeIdentifier" : [ {
-      "issuerReference" : {
-        "globalReference" : "d53e12f6",
-        "externalReference" : "partyA"
-      },
-      "assignedIdentifier" : [ {
-        "identifier" : {
-          "value" : "166555",
-          "meta" : {
-            "scheme" : "http://www.swapswire.com/spec/2001/trade-id-1-0"
-          }
-        }
-      } ],
-      "meta" : {
-        "globalKey" : "ec318949"
-      }
-    }, {
-      "issuerReference" : {
-        "globalReference" : "7766c429",
-        "externalReference" : "partyB"
-      },
-      "assignedIdentifier" : [ {
-        "identifier" : {
-          "value" : "166555",
-          "meta" : {
-            "scheme" : "http://www.swapswire.com/spec/2001/trade-id-1-0"
-          }
-        }
-      } ],
-      "meta" : {
-        "globalKey" : "ec31fda8"
-      }
-    } ],
-    "tradeDate" : {
-      "value" : "2009-01-27",
-      "meta" : {
-        "globalKey" : "3ec85b"
-      }
-    },
-    "tradableProduct" : {
-      "product" : {
-        "contractualProduct" : {
-          "productTaxonomy" : [ {
-            "source" : "ISDA",
-            "productQualifier" : "EquityOption_ParameterReturnVariance_Index"
```

Comment:
No longer qualifies due to MULTIPLE_MATCHES: EquityOption_PriceReturnBasicPerformance_SingleName,EquityOption_ParameterReturnVariance_Index